spec/tests/PlayerEvents.spec.ts

```diff
@@ -90,7 +70,7 @@ describe('player event tests', () => {
           playerMock.eventEmitter.firePlayEvent();
           // TODO: check if delayed is testable
           setTimeout(function () {
-            expect(playerStateMock.setPlayerState).toHaveBeenCalledWith(Conviva.PlayerStateManager.PlayerState.BUFFERING);
+            expect(convivaVideoAnalytics.reportPlaybackMetric).toHaveBeenCalledWith(Conviva.Constants.Playback.PLAYER_STATE, Conviva.Constants.PlayerState.STOPPED);
```

Comment:
@gregrealeyes Wouldn't this be BUFFERING not STOPPED?

Reply:
I am going to remove this test, we don't get a buffering event here. we only get a stopped then a seeked for some reason

Reply:
ok, that's weird